lib/matplotlib/projections/polar.py

```diff
@@ -287,6 +287,9 @@ class ThetaLocator(mticker.Locator):
     locations of every 45 degrees are returned.
     """
 
+    def __repr__(self):
+        return f"{self.__class__.__name__}(base={self.base.__repr__()})"
```

Comment:
These `__repr__` should all be:
```suggestion
        return f"{self.__class__.__name__}(base={self.base!r})"
```